content/docs/guides/iframes@es.md

```diff
@@ -0,0 +1,67 @@
+---
+$title: Incluir Iframes
+$order: 2
+$category: Develop
+components:
+  - iframe
+toc: true
+---
+[TOC]
+
+Aprenda cómo mostrar contenido multimedia en sus páginas y cómo utilizar iframes
+para mostrar contenido avanzado fuera de las limitaciones de AMP.
+
+## Lo básico
+
+Muestra un iframe en tu página usando el elemento 
+[`amp-iframe`](/es/docs/reference/components/amp-iframe.html).
+
+Los iframes son especialmente útiles en AMP para mostrar contenido no
+soportado en el contexto de la página principal, como el contenido que requiere JavaScript.
+
+Requerimientos del elemento `amp-iframe`:
+
+* Debe ser al menos de **600px** o **75%** de la primera ventana de visualización lejos de la parte superior.
+* Solo puede solicitar recursos a través de HTTPS, y no deben estar en el mismo origen
```

Comment:
This is because in the file in english the Line break is present too.
Please take a look here: https://github.com/SidVal/docs/blame/fe9362540df0c66b765f848742eb5ae648a3f303/content/docs/guides/iframes.md#L26